projects/com.oracle.truffle.llvm.parser.base/src/com/oracle/truffle/llvm/parser/base/facade/NodeFactoryFacade.java

```diff
@@ -280,7 +279,7 @@ LLVMNode createSwitch(LLVMExpressionNode cond, int defaultLabel, int[] otherLabe
      * @return a function root node
      */
     RootNode createFunctionStartNode(LLVMExpressionNode functionBodyNode, LLVMNode[] beforeFunction, LLVMNode[] afterFunction, SourceSection sourceSection, FrameDescriptor frameDescriptor,
-                    FunctionHeader functionHeader);
+                    FunctionDeclaration functionHeader);
```

Comment:
I think you are right. Fixed https://github.com/graalvm/sulong/pull/516/commits/ff98b45dc1e66cc01196eb41512e22aa6e12adf9